modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// */openshift_pipelines/installing-pipelines.adoc
+[id="op-installing-pipelines-operator-in-web-console_{context}"]
+= Installing the {pipelines-title} Operator in web console
+
+You can install {pipelines-title} using the Operator listed in the {product-title} OperatorHub. When you install the {pipelines-title} Operator, the custom resources required for the Pipelines configuration are automatically installed along with the Operator.
+
+[discrete]
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Operators->OperatorHub*.
+
+. Use the *Filter by keyword* box to search for _{pipelines-title} Operator_ in the catalog. Click the *OpenShift Pipelines Operator provided by Red Hat* tile.
++
+image::op-installed-tile.png[]
+
+. Read the brief description about the Operator on the *{pipelines-title} Operator* page. Click *Install*.
++
+image::op-install-subscription.png[]
+
+. On the *Create Operator Subscription* page:
+
+  .. Select *All namespaces on the cluster (default)* for the *Installation Mode*. This mode installs Operator in the default `openshift-operators` namespace, which enables the Operator to watch over, and be made available to all namespaces in the cluster.
+
+  .. Select *Automatic* for  *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the the Operator Lifecycle Manager (OLM). If you select *Manual*; when a new version of the Operator is available, you will need to manually approve the OLM update request to update to the new version.
+  .. Select an *Update Channel*.
+    *** The `ocp-4.4` channel enables installation of the latest stable release of the {pipelines-title} Operator.
+    *** The `preview` channel enables installation of the latest preview version of the {pipelines-title} Operator, which may contain features that are not yet available from the 4.4 update channel.
```

Comment:
I was asked by Pavol to specify the channel name, see https://github.com/openshift/openshift-docs/pull/20746#discussion_r409446648 and https://github.com/openshift/openshift-docs/pull/20746#issuecomment-611512131

I agree that we should make it generic here. Let me add  **ocp-<4-x>**. Will see what the OCP folks have to say

Reply:
I agree that it would remove maintenance burden and it is clear enough for me. Beware of another string "4.4" in this sentence, this needs to be updated for each release, too. Alternative would be to use a variable, I guess that some is already there because it's same as version of OpenShift.

Reply:
done